deepchecks/tabular/checks/model_evaluation/performance_disparity_report.py

```diff
@@ -0,0 +1,421 @@
+
+    def _add_differences_traces(self, sub_visual_df, fig, row=1, col=1):
+        sub_visual_df = sub_visual_df.sort_values("_diff").head(self.max_subgroups_per_category_to_display)
+        sub_visual_df = sub_visual_df.sort_values("_diff", ascending=False)
+        for _, df_row in sub_visual_df.iterrows():
+            subgroup = df_row[self.protected_feature]
+            baseline = df_row["_baseline"]
+            score = df_row["_score"]
+            color = "orangered" if df_row["_diff"] < 0 else "limegreen"
+            extra_label = "<extra></extra>"
+
+            fig.add_trace(
+                go.Scatter(
+                    x=[score, baseline],
+                    y=[subgroup, subgroup],
+                    hovertemplate=[
+                        "%{y}: %{x} (group size: " + str(df_row["_count"]) + ")" + extra_label,
+                        "baseline: %{x} (group size: " + str(df_row["_baseline_count"]) + ")" + extra_label,
+                    ],
+                    marker=dict(
+                        color=["white", "#222222"], symbol=0, size=6, line=dict(width=[2, 2], color=[color, color])
+                    ),
+                    line=dict(color=color, width=8),
+                    opacity=1,
+                    showlegend=False,
+                    mode="lines+text+markers",
+                    cliponaxis=False,
+                ),
+                row=row,
+                col=1,
+            )
+
+    def make_largest_difference_figure(self, scores_df: pd.DataFrame):
+        """
+        Create "largest performance disparity" figure.
+
+        Parameters
+        ----------
+        scores_df : DataFrame
+            Dataframe of performance scores, as returned by `make_scores_df()`, disaggregated by feature and control_feature, and with average scores for each control_feature level. Columns named after `feature` and (optionally) `control_feature` are expected, as well as columns named "_scorer", "_score", "_baseline", and "_count".
+
+        Returns
+        -------
+        Figure
+            Figure showing subgroups with the largest performance disparities.
+        """
+
+        visual_df = scores_df.copy().dropna()
+
+        has_control = self.control_feature is not None
+        has_model_classes = "_class" in visual_df.columns.values
+
+        subplot_grouping = []
+        if has_control:
+            subplot_grouping += [self.control_feature]
+        if has_model_classes:
+            subplot_grouping += ["_class"]
+        # Get distinct subplot categories with the largest observed differences
+        if len(subplot_grouping) > 0:
+            subplots_categories = (
+                visual_df.sort_values("_diff", ascending=True)[subplot_grouping]
+                .drop_duplicates()
+                .head(self.max_categories_to_display)
+            )
+            rows = len(subplots_categories)
+            if rows == 0:
+                raise DeepchecksProcessError("Nothing to display.")
```

Comment:
This happens when you are grouping by control_feature or model_class, but the subgroups are too small to have any computed scores. I replaced the error with the following:

```python
if len(visual_df) == 0:
    # Return text figure: "No scores to display"
    fig = go.Figure()
    fig.update_layout(xaxis_visible=False, yaxis_visible=False)
    fig.add_annotation(
        text="No scores to display",
        xref="paper",
        yref="paper",
        showarrow=False,
        font=dict(size=28),
    )
    return fig
```

Reply:
This is what it looks like: 
![screenshot](https://user-images.githubusercontent.com/784901/209153111-f54e1cf1-c917-4b19-97f1-dffa2ebc74ab.png)

Reply:
1. You can just have text rather than a plotly graph 
2. Perhaps tell the user that we're not displaying anything because there are not enough samples.

Reply:
Replaced the plotly Figure with a text string: f"No scores to display. Subgroups may be smaller than the minimum size of {self.min_subgroup_size}."